Auth0/Response.swift

```diff
@@ -57,7 +57,7 @@ struct Response<E: Auth0Error> {
             return json
         }
         // This piece of code is dedicated to our friends the backend devs :)
-        if response.url?.lastPathComponent == "change_password" {
+        if response.url?.lastPathComponent == "change_password" || response.url?.lastPathComponent == "revoke" {
```

Comment:
That if is a hack to avoid blowing up for change_password since it does not follow the rules of content negotiation. I'd avoid adding extra hacks like this, I though we have a way to tell there is no response or I will ignore the response (Maybe in Guardian.swift we did something like it)